### PR TITLE
Terraform: Update GPG_KEY_SERVERS

### PR DESCRIPTION
**Pull request by samruddhikhandale**
Addresses - https://github.com/devcontainers/features/issues/323

### CODE REVIEW

File: src/terraform/install.sh [hunk 1]
```diff
@@ -44,26 +44,9 @@ if [ "$(id -u)" -ne 0 ]; then
     exit 1
 fi
 
-# Get central common setting
-get_common_setting() {
```
Comment (samruddhikhandale) [inline src/terraform/install.sh:48]: @Chuxel Any concerns with removing this? 
We already have the correct variables set in this `install.sh` file.

File: src/terraform/install.sh [hunk 2]
```diff
@@ -26,7 +26,7 @@ TERRAFORM_DOCS_SHA256="${TERRAFORM_DOCS_SHA256:-"automatic"}"
 
 TERRAFORM_GPG_KEY="72D7468F"
 TFLINT_GPG_KEY_URI="https://raw.githubusercontent.com/terraform-linters/tflint/master/8CE69160EB3F2FE9.key"
-GPG_KEY_SERVERS="keyserver hkp://keyserver.ubuntu.com:80
```
Comment (samruddhikhandale) [inline src/terraform/install.sh:29]: Many folks have reported that the fetching GPG key from `hkp://keyserver.ubuntu.com:80` stuck for them and works with `hkp://keyserver.ubuntu.com`. 

https://github.com/devcontainers/features/issues/323

File: src/terraform/install.sh (hunk 1 above)
Comment (samruddhikhandale) [reply src/terraform/install.sh:48]: If we happen to need this `settings.env` file, we should think of migrating it from the old repo.

**Review by joshspicer — APPROVED** [review]

**Comment by seBasKov** [comment]
Hi samruddhikhandale, I noticed after this change my terraform feature install hangs at downloading the GPG key. I was referring to major version 1 as a result it picks up your change 1.1.1. 

My issue is solved using version 1.1.0 again. 

Might be worth to have a look as it seems that this bugfix introduces a new one.

**Pull request merged**